m/martiora.md

```diff
@@ -0,0 +1,9 @@
+# Martiora Freedom
```

Comment:
The filename for this artist looks wrong. Shouldn't it be `martiora-freedom`?

Reply:
This is the new one, please could you check?